tests/test_fed_avg_optimizer.py

```diff
@@ -154,3 +161,12 @@ def test_optimizer_schedules_communication_after_training(initialization_payload
     assert event_type == ActionableEventTypes.SCHEDULE_JOBS.name
     for job in job_arr:
         assert job.job_type == JobTypes.JOB_COMM.name
+
+# def test_optimizer_waits_to_average_until_trained(initialization_payload, train_dmlresult_obj):
+#     optimizer = FederatedAveragingOptimizer(initialization_payload)
+#     args = {
+#         TxEnum.KEY.name: MessageEventTypes.NEW_WEIGHTS.name,
+#         TxEnum.CONTENT.name: serialize_job(train_dmlresult_obj.job)
+#     }
+#     nothing, nothing_ = optimizer.ask(RawEventTypes.NEW_MESSAGE.name, args)
+#     assert False
```

Comment:
Did you mean to leave this commented out?

Reply:
yes